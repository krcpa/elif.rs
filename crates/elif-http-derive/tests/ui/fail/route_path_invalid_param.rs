use elif_http_derive::get;

#[get("/users/{}")]  // Empty parameter name
fn get_user() -> String {
    "user".to_string()
}