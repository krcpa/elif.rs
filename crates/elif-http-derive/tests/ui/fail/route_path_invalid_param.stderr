error: Invalid route path '/users/{}': empty parameter name '{}'. Hint: Use format like '/users/{id}' with proper parameter syntax.
 --> tests/ui/fail/route_path_invalid_param.rs:3:7
  |
3 | #[get("/users/{}")]  // Empty parameter name
  |       ^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/fail/route_path_invalid_param.rs:6:2
  |
6 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/fail/route_path_invalid_param.rs`
